strings/traumas.json

```diff
@@ -2,6 +2,7 @@
     "brain_damage": [
         "@pick(semicolon)IM A PONY NEEEEEEIIIIIIIIIGH",
         "without oxigen blob don't evoluate?",
+		"THE FUCKING CHIEF ENGEENIR STUNNED ME AND PUT ME OUT OF ENGEENERING",
```

Comment:
Wrong indentation

Reply:
It doesn't show up that way for me

Reply:
You're using tabs instead of spaces for indentation is the issue. This file is space indented